lib/backend.ts

```diff
@@ -32,10 +33,9 @@ export interface SessionHandler {
   /**
    * Resolves the operator from the name; backend specific
```

Comment:
nit: I think the comment would be - resolve the operator from the name and the opset version ?

Reply:
will fix